frontend/packages/core/src/NPS/feedback.tsx

```diff
@@ -185,21 +204,42 @@ const NPSFeedback = (opts: FeedbackOptions) => {
         container
         direction="row"
         alignItems="center"
-        justify="center"
-        style={{ padding: "16px" }}
+        style={{ padding: wizardOrigin ? "16px" : "32px" }}
       >
-        <MuiGrid item xs={6}>
-          <Typography variant="subtitle3">{survey.prompt}</Typography>
+        <MuiGrid item xs>
+          <Typography variant={wizardOrigin ? "subtitle3" : "subtitle2"}>
+            {survey.prompt}
+          </Typography>
         </MuiGrid>
-        <MuiGrid item xs={6} style={{ display: "flex", justifyContent: "space-around" }}>
-          <EmojiRatings ratings={survey.ratingLabels} setRating={setSelected} />
+        <MuiGrid
+          item
+          xs={wizardOrigin ? 6 : 12}
+          style={{ margin: "8px 0px", display: "flex", justifyContent: "space-around" }}
+        >
+          <EmojiRatings
+            ratings={survey.ratingLabels}
+            setRating={setSelectedEmoji}
+            placement={wizardOrigin ? "top" : "bottom"}
+            size={wizardOrigin ? "small" : "large"}
```

Comment:
i know the figma spec is the large size, but does it maybe look too big? maybe we should use the `medium` variant (32x32)? but could just be me

Reply:
Yeah its a bit large, I'll adjust it to use medium


![Screen Shot 2022-01-21 at 3 53 26 PM](https://user-images.githubusercontent.com/8338893/150614523-d0c8ed66-f9e7-4f85-86b7-36e21741b175.png)